pipeline/console/targets/pipelines.py

```diff
@@ -55,6 +59,18 @@ def get_parser(command_parser: "_SubParsersAction[ArgumentParser]") -> None:
         help="Pipeline name.",
         type=str,
     )
+    get_parser.add_argument(
+        "--skip",
+        "-s",
+        help="Number of pipelines to skip in paginated set.",
+        type=int,
+    )
+    get_parser.add_argument(
+        "--limit",
+        "-l",
+        help="Total number of pipelines to fetch in paginated set.",
+        type=int,
+    )
```

Comment:
looks good, just wonder if in the main documentation for the command we can add an example since it might not be obvious how to use this?